styles/globals.scss

```
@@ -11,6 +11,9 @@
   line-height: $font-base-line-height;
   scroll-behavior: smooth;
 
+  // variables
+  --header-height: 72px;
```

Instead of using CSS variables, in this case, would be better to use SASS variables
```suggestion
  $header-height: 72px;
```

Also, move it to the top of the file or to a proper file, for example 'layout.scss'

---

I tried using a sass variable but it doesn't work. https://www.alpharithms.com/sass-scss-variables-with-css-calc-functions-474422/